dimod/reference/samplers/exact_solver.py

```diff
@@ -22,12 +22,16 @@
 import numpy as np
 
 from dimod.core.sampler import Sampler
-from dimod.sampleset import SampleSet, as_samples
+from dimod.sampleset import SampleSet
 from dimod.core.polysampler import PolySampler
 from dimod.vartypes import Vartype
-from dimod.exceptions import SamplerUnknownArgWarning
 
-__all__ = ['ExactSolver', 'ExactPolySolver', 'ExactDQMSolver']
+from itertools import product
```

Comment:
Still want these above the others since they are standard library imports

Reply:
Apologies, I need to get my eyes checked apparently